src/components/features/forms/NewVendorForm.tsx

```diff
@@ -47,13 +62,30 @@ export default function NewVendorForm({
       />
 
       <Controller
-        name="isFinal"
+        name="markers"
+        defaultValue={defaultValues.markerIds?.map(String)}
         control={control}
         render={({ field }) => (
-          <FormControlLabel
-            control={<Checkbox {...field} checked={field.value} />}
-            label={translate('newVendor.labels.isFinal')}
-          />
+          <FormControl fullWidth>
+            <InputLabel id="demo-multiple-markers-label">
+              {translate('newVendor.labels.markers')}
+            </InputLabel>
+            <Select
+              {...field}
+              label={translate('newVendor.labels.markers')}
+              labelId="demo-multiple-markers-label"
+              id="demo-multiple-markers"
+              multiple
+              value={field.value || []}
+              onChange={(e) => field.onChange(e.target.value)}
+              renderValue={(selected) => (selected as string[]).join(', ')}>
+              {markers.map((marker: MarkerDto) => (
+                <MenuItem key={marker.id} value={marker.id?.toString()}>
+                  <ListItemText primary={marker.name} />
+                </MenuItem>
```

Comment:
we need to show the marker name and not id when we have selected something
![image](https://github.com/user-attachments/assets/98561a1f-6e7e-407b-95ff-a82df14d44e6)

You can check what Boyadzhiev has done in Zones Form to fix that.

Reply:
Fixed.